api/DOMTokenList.json

```diff
@@ -30,7 +30,7 @@
             "version_added": "11.5"
           },
           "safari": {
-            "version_added": "6"
+            "version_added": "5.1"
```

Comment:
Should "safari_ios" entries also be updated? Safari 5.1 + iOS 6 isn't a matching pair per https://github.com/mdn/browser-compat-data/pull/11299/files.